pekko-sample-cluster-kubernetes-java/src/main/resources/application.conf

```diff
@@ -33,4 +33,9 @@ pekko.management {
       example-ready = org.apache.pekko.cluster.bootstrap.demo.DemoHealthCheck
     }
   }
+  # Add these so the pods won't get killed.
+  http {
+    port = 8558
```

Comment:
You should set it to bind-port instead of port.

https://github.com/apache/pekko-management/blob/1b7a52ba277daec3130baee4bf29a2f0f55a51bb/management/src/main/resources/reference.conf#L33

If the user does not want to change, they just need to set it to pekko-management port in the k8s health check.(is 7626 not 8558)

https://github.com/apache/pekko-management/blob/1b7a52ba277daec3130baee4bf29a2f0f55a51bb/management/src/main/resources/reference.conf#L19